src/hooks/useTypefast.ts

```diff
@@ -39,12 +39,13 @@ export const useTypefast = () => {
   }, []);
 
   useEffect(() => {
-    let interval = 0;
+    let interval: NodeJS.Timeout;
     if (isStarted && seconds > 0) {
-      interval = setInterval(() => {
+      setInterval(() => {
```

Comment:
Bring back assignment due to bug

```suggestion
      interval = setInterval(() => {
```